web-console/5-components/order-from-catalog.md

```diff
@@ -54,51 +46,58 @@ There are 3 possible steps in this process: Plan, Configuration and Results.
 
-#### Catalog item has a multiple versions
+### Plan Step (Optional)
+This step should be shown if there are 1+ plans.
+
+#### Single Plan
 ![template](img/plan-step.png)
 
-#### Implementation Details
-- This step is optional
-- Right column of the content panel includes:
-	- If there are > 1 plans, this step should be added
-	- If multiple versions exist, include version selection on this panel
-		- Version is associated with Builder Images not catalog items
-		- The first version should be selected by default
-	- Each plan should be presented as a radio button option, by the plan title.  
-		- If the plan has a description, it should be listed under the title
-		- The first plan should be selected by default.
+For each plan, display:
+- Plan title
+	- metadata.displayName from ![Plan Metadata Fields](https://github.com/openservicebrokerapi/servicebroker/blob/master/profile.md#plan-metadata-fields)
+- Optional description
+- Features
+	- if the plan has 1+ features, show a **Features** title following by a list of features
+	- metadata.bullets from ![Plan Metadata Fields](https://github.com/openservicebrokerapi/servicebroker/blob/master/profile.md#plan-metadata-fields)
+- Pricing
+	- if the plan has 1+ pricing options, show a **Pricing** title following by a list of costs
+	- amount from ![Cost Object of the Plan Metadata Fields](https://github.com/openservicebrokerapi/servicebroker/blob/master/profile.md#plan-metadata-fields)
+	- unit from ![Cost Object of the Plan Metadata Fields](https://github.com/openservicebrokerapi/servicebroker/blob/master/profile.md#plan-metadata-fields)
+
 - Buttons associated with this step
-	- Cancel, Back & Next buttons should be displayed
-	- Cancel & Next should always be enabled
-	- Back should be disabled
+	- Cancel, Back & Next buttons should be displayed & enabled
 
-### Step 2 - Configuration
 
-#### Catalog item has multiple versions and version was not included on the plan step
-![template](img/config-step.png)
+#### Multiple Plans
 
-#### Catalog item has single versions OR version was included on the plan step
-![template](img/config-step-no-version.png)
+![template](img/plan-singleplan-step.png)
+- Each plan should be presented as a radio button option, by the plan title.  
+- The first plan should be selected by default.
 
-#### Catalog item shows advanced options
-![template](img/config-step-hide-advanced.png)
-
-#### User views available projects in context of this flow
-![template](img/config-step-select-project.png)
 
-#### User creates a new project in context of this flow
-![template](img/config-step-create-project.png)
+### Configuration Step
 
 #### Implementation Details
 - This step is mandatory
-- Right column of the content panel includes:
+- Content panel includes:
 	- Optional version selection
 		- If there aren’t more than one plan, and there are more than one version available, include a Version list.
```

Comment:
This is confusing.

Reply:
which piece?  I don't think i changed this :)